apps/api/src/models/ApplicationData.py

```diff
@@ -52,21 +54,21 @@ class BaseApplicationData(BaseModel):
 class BaseMentorApplicationData(BaseModel):
     model_config = ConfigDict(str_strip_whitespace=True, str_max_length=254)
 
-    experienced_technologies: str
-    pronouns: str
+    experienced_technologies: Optional[List[str]] = []
+    pronouns: Optional[List[str]] = []
```

Comment:
Suggestion: If we are specifying a default value of an empty list, these fields do not need to be possibly `None`